netmiko/huawei/huawei_ssh.py

```diff
@@ -80,3 +80,34 @@ def set_base_prompt(self, pri_prompt_terminator='>', alt_prompt_terminator=']',
         log.debug("prompt: {0}".format(self.base_prompt))
 
         return self.base_prompt
+
+
+class HuaweiVrpv8SSH(HuaweiSSH):
+
+    def commit(self, comment='', delay_factor=1):
+        """
+        Commit the candidate configuration.
+
+        Commit the entered configuration. Raise an error and return the failure
+        if the commit fails.
+
+        default:
+           command_string = commit
+        comment:
+           command_string = commit comment <comment>
+
+        """
+        delay_factor = self.select_delay_factor(delay_factor)
+        error_marker = 'Failed to generate committed config'
+        command_string = 'commit'
+
+        if comment:
+            command_string += ' comment "{}"'.format(comment)
+
+        output = self.config_mode()
+        output += self.send_command_expect(command_string, strip_prompt=False,
+                                           strip_command=False, delay_factor=delay_factor)
```

Comment:
You probably should exit config mode after you do this `self.exit_config_mode()`.

Reply:
Thanks  @ktbyers 
have updated with self.exit_config_mode(), the patch is verified with ironic baremetal with huawei switch ce5800